common/config/rush/pnpm-lock.yaml

```diff
@@ -260,32 +695,32 @@ packages:
     hasBin: true
     resolution:
       integrity: sha512-sXobUDKsKx2apisLFhk5gxBPBfnCbM31hpmQwqHAbwZ7ak4Sj7I+OcN41hSwbIQksZnk2OSbu+WElEehHiS+xA==
-  /@microsoft/gulp-core-build-mocha/3.8.5:
+  /@microsoft/gulp-core-build-mocha/3.8.8:
     dependencies:
-      '@microsoft/gulp-core-build': 3.15.2
+      '@microsoft/gulp-core-build': 3.15.5
       '@types/node': 10.17.13
       glob: 7.0.6
       gulp: 4.0.2
       gulp-istanbul: 0.10.4
       gulp-mocha: 6.0.0
     dev: false
     resolution:
-      integrity: sha512-d8B83x4pPTnDGjwno8D30Iier4Vn/tNTyV3kL3FxvwWF9ro6cP5nz0LythFCdxnkGivupIQa1ShmtI2WToQ7Cg==
-  /@microsoft/gulp-core-build-typescript/8.4.5:
+      integrity: sha512-yCiqH/owOrkpfr5t8OVm8Pc/SYbbkJc81M7P7v7oa9MbmoQlaFphcHBIgxAl7/Tc8xtqovQ64gnCKIqfi3NLAg==
+  /@microsoft/gulp-core-build-typescript/8.4.10:
     dependencies:
-      '@microsoft/gulp-core-build': 3.15.2
-      '@rushstack/node-core-library': 3.19.4
+      '@microsoft/gulp-core-build': 3.15.5
+      '@rushstack/node-core-library': 3.19.7
       '@types/node': 10.17.13
       decomment: 0.9.2
       glob: 7.0.6
       glob-escape: 0.0.2
       resolve: 1.8.1
     dev: false
     resolution:
-      integrity: sha512-s3HOpXfX2byc49kTgs9kOqmgvjxkcJcyvUWV5vZkwRJ6nlWVbAu3lL+5rjr35dmNTL1Is8FusR3YXaLdRHZ+Rw==
-  /@microsoft/gulp-core-build/3.15.2:
+      integrity: sha512-vy8FD8utQW0vzGsU4Gpr9yH1RUQgWiPdbQWS470Epa47hBlfZh4bXnHXp7dtsZ/q2oN0yMTZSzIS7Qx8B+Hpyw==
+  /@microsoft/gulp-core-build/3.15.5:
```

Comment:
@octogonz Do you know know why 'jest' would remain at version `23.6.0` even though I have upgraded it to `25.1.0` in gulp-core-build's `package.json`? I have been trying to figure this out but didn't get anywhere.

Reply:
@sachinjoseph  Yes, there are two `gulp-core-build`'s in your **pnpm-lock.yaml** file.

`'file:projects/gulp-core-build.tgz':` is the local project which I do see as having been upgraded to `25.1.0` in your shrinkwrap file.

Whereas `/@microsoft/gulp-core-build/3.15.5` is the previously published package from NPM, which is being pulled in due to the `cyclicDependencyProjects` in **rush.json** which installs `@microsoft/node-library-build` which relies on the published GCB.

So your shrinkwrap file is probably correct.  But in order to fully upgrade Jest, you will have to merge this PR, publish the new packages, and then do another PR to upgrade the cyclic dependencies.  (See PR https://github.com/microsoft/rushstack/pull/1594 and PR https://github.com/microsoft/rushstack/pull/1607 for examples.)

Reply:
I see, that makes sense. Thanks @octogonz !

Reply:
My goal is it upgrade jest, jest-cli etc. to the latest.

Reply:
FYI historically Jest upgrades ran into problems because Jest had loose dependencies that rely on the "hoisting" installation strategy used by Yarn.  So upgrading Jest sometimes required workarounds in `common-versions.json` or `pnpmfile.js`.  Hopefully it's better behaved by now.

Reply:
Thank you. I am actually running into a few issues while I am trying to upgrade jest-cli. I will see if they can be resolved using pnpmfile or common-versions.